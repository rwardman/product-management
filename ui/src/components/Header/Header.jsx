import React from "react";

import Logo from "../Icons/Logo";

import * as Styled from "./Header.styles.js";

const Header = () => {
  return (
    <Styled.Container>
      <Styled.LogoContainer>
        <Logo />
      </Styled.LogoContainer>
      <h1>Product Management</h1>
    </Styled.Container>
  );
};

export default Header;
