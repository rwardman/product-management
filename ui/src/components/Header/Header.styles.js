import styled from "styled-components";

const Container = styled.div`
  display: flex;
  margin: 20px;
  border-bottom: 1px solid #d8d8d8;
`;

const LogoContainer = styled.div`
  margin-right: 20px;
`;

export { Container, LogoContainer };
