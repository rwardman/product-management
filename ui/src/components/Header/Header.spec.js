import renderer from "react-test-renderer";
import Header from "./Header";

describe("When displaying the header component", () => {
  it("should be displayed correctly", () => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
