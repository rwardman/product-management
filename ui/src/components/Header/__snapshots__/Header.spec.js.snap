// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`When displaying the header component should be displayed correctly 1`] = `
<div
  className="sc-bdfBwQ zDYku"
>
  <div
    className="sc-gsTCUz jXrPCQ"
  >
    <svg
      height="60"
      version="1.0"
      viewBox="0 0 512 512"
      width="60"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M0 256v256h512V0H0v256zm271-136c22.7 2.2 37.6 8.7 51.1 22 7 6.9 9.4 10.1 12.8 17 6.3 13.1 8.3 22 8.3 38 0 10.5-.4 15.2-2 21.1-7.1 26.4-23.2 42.7-49.7 50.4-11.5 3.3-28.1 4.5-62.7 4.5H201l-.2 60.2-.3 60.3-15.7.3-15.8.3V119h46.3c25.4 0 50.5.5 55.7 1z"
        fill="#1c1970"
      />
      <path
        d="M201 196.4v48.8l36.3-.4c39.7-.5 40.8-.6 53-7.1 14.7-7.7 22.9-24.5 21.3-43.7-1.8-23-14.4-38-36.8-43.6-4.9-1.2-14-1.7-40-2.1l-33.8-.6v48.7z"
        fill="#1c1970"
      />
    </svg>
  </div>
  <h1>
    Product Management
  </h1>
</div>
`;
